tools/build_rules/llvm/cmakedefines/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["cmakedefines.go"],
+    importpath = "github.com/shahms/cmaketobzl/tools/cmakedefines",
```

Comment:
Is that the canonical location?

Reply:
That was where it was during development, but the current location is canonical for now (although fairly academic as nothing should reference it by that name).